### ISSUE TITLE
[Feature]: Implement unHead() to create dynamic metadata

### ISSUE THREAD

**Issue by itsalaidbacklife** [cuttle-cards/cuttle#462]
### Feature Summary

Currently, as of #458, we have hard-coded `<meta>` tags in the `<head>` of `index.html`. This defines basic metadata for the site, but it could be made dynamic to allow for variable use and page-specific metadata.

### Detailed Description

The [unHead lib](https://unhead.harlanzw.com/guide/getting-started/setup) has become the standard for managing dynamic metadata in vue applications (as well as other frameworks). We should implement in in Cuttle so that we can:
1. Take the hard-coded meta data defined in `index.html` `<meta>` tags and variablize it (thereby removing the duplicate data used in the `og` and `twitter` card specs
2. Set up the basis for establishing page-specific metadata e.g. identifying the stats of a particular season

This is a follow up on the initial definition of metadata created in #458.

It should involve defining variables to manage any overlap in the existing metadata found in `index.html` in addition to the setup of the possibility for page-specific metadata

**Comment by Dxuian** [comment]
still open ?

**Comment by itsalaidbacklife** [comment]
> still open ?

Yes @Dxuian this issue is still open and would make for a useful contribution

**Comment by KaroK365** [comment]
Can you assign it to me  😊

**Comment by ryuga001** [comment]
Can you assign it to me ?
